### PR TITLE
Feat/connect backend service

### PR DESCRIPTION
**Pull request by ckouder**

### CODE REVIEW

**Review by bodom0015 — APPROVED** [review]
This looks amazing! Great work, @ckouder!
Tested by viewing the following previously-completed jobs :+1:

NovoStoic - Overall Stoichiometry (optStoic): https://novostoic.frontend.staging.mmli1.ncsa.illinois.edu/overall-stoichiometry/result/6aebe3616c2f44579b7844feb916ab69
NovoStoic - Pathway Search (novoStoic aka pathways): https://novostoic.frontend.staging.mmli1.ncsa.illinois.edu/pathway-search/result/368d69df31994747a497c76afc13f98e
NovoStoic - Thermodynamical Feasibility (dGPredictor): https://novostoic.frontend.staging.mmli1.ncsa.illinois.edu/thermodynamical-feasibility/result/608e5717bdd74c678077356676abe78f
NovoStoic - Enzyme Selection (EnzRank): https://novostoic.frontend.staging.mmli1.ncsa.illinois.edu/enzyme-selection/result/fe60759487574931afa31a5e8f544727

File: chart/values.staging.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ ingress:
     traefik.ingress.kubernetes.io/router.tls: "true"
 
 controller:
-  image: moleculemaker/novostoic-frontend:latest
+  image: moleculemaker/novostoic-frontend:pr-34
```
Comment (bodom0015) [inline chart/values.staging.yaml:10]: You might want to revert this change before merging :+1:
```suggestion
  image: moleculemaker/novostoic-frontend:latest
```

**Comment by bodom0015** [comment]
@ckouder for clarity, does this PR contain #31 and/or #30?

Or do those PRs need to be reviewed separately 🤔 let me know if I need to take a look at those previous iterations!

**Comment by ckouder** [comment]
> @ckouder for clarity, does this PR contain #31 and/or #30?
> 
> Or do those PRs need to be reviewed separately 🤔 let me know if I need to take a look at those previous iterations!

@bodom0015 Thank you for reviewing! This pr contains them all!
